src/components/chart/_macro.njk

```diff
@@ -91,7 +92,12 @@
                         tabindex="0"
                         role="region"
                         aria-label="chart container"
+                        aria-describedby="{{ instructionsId }}"
                     >
+                        <div id="{{ instructionsId }}" class="ons-u-vh">
+                            Use the Tab key to move focus into the chart. Once inside, use the arrow keys to navigate between data points.
+                            As you move, tooltips will be announced to describe each point.
+                        </div>
```

Comment:
This shouldn't be hardcoded. Have a look at the autosuggest to see how its been done

Reply:
updated it